src/Symfony/Component/ClassLoader/ClassMapGenerator.php

```diff
@@ -134,7 +134,7 @@ private static function findClasses($path)
                     }
 
                     if ($isClassConstant) {
-                        continue;
+                        continue 2;
```

Comment:
Break would be nicer, agree. Not having default is quite bad practice actually, because not clear what shall happen with non-handled values.